dashboard/lib/dynamic_config/loaders/gatekeeper_loader.rb

```diff
@@ -0,0 +1,62 @@
+# A script to update the Gatekeeper settings
```

Comment:
This seems to be the same as dcdo_loader, perhaps this is a renaming and you want to get rid of the other one?